cea/interfaces/dashboard/inputs/static/js/map.js

```diff
@@ -0,0 +1,274 @@
+const {DeckGL, GeoJsonLayer} = deck;
+var deckgl;
+var jsonStore = {};
+// Make sure the layers maintain their order
+var layers = [new GeoJsonLayer({id:'streets'}), new GeoJsonLayer({id:'dh_networks'}),
+    new GeoJsonLayer({id:'dc_networks'}), new GeoJsonLayer({id:'zone'}), new GeoJsonLayer({id:'district'})];
+var extruded = false;
+var currentViewState;
+
+const jsonUrls = {
+    'zone': 'http://localhost:5050/inputs/geojson/zone',
```

Comment:
use relative url's here, since the dashboard should also work with other ports / hosts (like, when we move to the cloud)

Reply:
ah yes, my mistake